scripts/us_fed/treasury_constant_maturity_rates/extract_data.py

```diff
@@ -0,0 +1,41 @@
+USAGE = '''
+python3 extract_data.py [path]
+    "path": Path to the raw csv containing rates at all maturities.
+
+Example: python3 extract_data.py FRB_H15.csv
+    This will extract the data out of the raw csv and store it in
+    "treasury_constant_maturity_rates.csv".
+    The output table has the same number of columns as the number of constant
+    maturities provided and an extra column for dates.
+    "date" column is of the form "YYYY-MM-DD".
+    The other interest rate columns are numeric.
+'''
+
+
+import sys
+
+from maturity import MATURITIES
+import pandas as pd
```

Comment:
do you need a requirements.txt for pandas

Reply:
Added

Reply:
I didn't see a requirements.txt